app/views/alchemy/admin/pages/_sitemap.html.erb

```diff
@@ -18,7 +18,17 @@
    Alchemy.Sitemap.init(
      '<%= alchemy.tree_admin_pages_path %>',
      <%= @page_root.id %>,
-     <%= !!@sorting %>
+     <%= full %>
+     <% if @sorting  %>
+     ,function () {
+       Alchemy.PageSorter.init();
+     }
+     <% end %>
+     <% if @link %>
+     ,function () {
+       window.dialog.attachEvents();
```

Comment:
As mentioned above, please use `Alchemy.currentDialog()` here to get the current dialog instance.